src/LinAlg/hiopLinSolverSparseCUSOLVER.cpp

```diff
@@ -749,7 +754,7 @@ namespace hiop
     assert(CUSOLVER_STATUS_SUCCESS == sp_status_);
 
     buffer_size_ = size_M_;
-    cudaFree(d_work_);
+    //cudaFree(d_work_);
```

Comment:
Remove rather than comment out this line:
```suggestion
```

Reply:
okay, changed.